pkg/tiller/release_server.go

```diff
@@ -525,11 +525,12 @@ func (s *ReleaseServer) performRollback(currentRelease, targetRelease *release.R
 	return res, nil
 }
 
-func (s *ReleaseServer) performKubeUpdate(currentRelease, targetRelease *release.Release, recreate bool, timeout int64, shouldWait bool) error {
+func (s *ReleaseServer) performKubeUpdate(currentRelease, targetRelease *release.Release, force bool, recreate bool, timeout int64, shouldWait bool) error {
+	log.Printf("perform: force: %v", force)
```

Comment:
Not sure how much value this adds.  Can it be removed or moved somewhere more pertinent